zerver/migrations/0018_merge.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
This "merge" migration got generated because the original PR for starred streams is very old and used migration 6 for adding the new "starred" feature; and migration 6 had already been used.

If you rm all the migrations and rerun `./manage.py makemigrations` you'll get a much more normal single migration adding the "starred" field.